sound/soc/sof/intel/cnl.c

```diff
@@ -143,9 +142,7 @@ irqreturn_t cnl_ipc_irq_thread(int irq, void *context)
 		msg = hipctdr & CNL_DSP_REG_HIPCTDR_MSG_MASK;
 		msg_ext = hipctdd & CNL_DSP_REG_HIPCTDD_MSG_MASK;
 
-		dev_vdbg(sdev->dev,
-			 "ipc: firmware initiated, msg:0x%x, msg_ext:0x%x\n",
-			 msg, msg_ext);
+		trace_sof_intel_ipc_firmware_initiated(sdev, msg, msg_ext);
```

Comment:
@nklayman, do you have example on how the trace looks like from
`trace_sof_intel_ipc_firmware_response(sdev, msg, msg_ext);`
and
`trace_sof_intel_ipc_firmware_initiated(sdev, msg, msg_ext);`

Reply:
`response` is printing 0 for both the `msg` and `msg_ext`, while `initated` always prints 0 for `msg_ext`:

```
irq/133-AudioDS-525     [000] .....   215.486188: sof_intel_ipc_firmware_initiated: device_name=0000:00:1f.3 msg=0x10020000 msg_ext=0x0

irq/133-AudioDS-525     [000] .....   147.158905: sof_intel_ipc_firmware_response: device_name=0000:00:1f.3 msg=0x0 msg_ext=0x0
```